content/news/core-monthly/2023/core-monthly-2023-05-01-2023-05-31.md

```diff
@@ -0,0 +1,404 @@
+
+You can find the RC1 [release notes here]({{< relref "/news/releases/prestashop-8-1-rc1-release" >}}).
+
+### PrestaShop 9 news!
+
+Work on PrestaShop 9 has already started. As of now we can see a lof of cleaning in the core, removing deprecated functions, upgrading dependencies, but also some new existing features coming. Let's do a little recap.
+
+**PHP 8.1 as a minimum compatible version in PrestaShop 9.0**
+
+You read that right. After discussing internally and with the community, the leadership council decided to bump the minimum compatible version of PHP to version 8.1. It's great news for developers who want to use the latest features of PHP in their modules. It's also great news for the PrestaShop core, as we can use the some of the latest features of PHP in the core. We'll also be able to use the latest versions of our dependencies, making the core more secure and faster. You can read [full announcement about the decision]({{< relref "/news/posts/2023/prestashop-9-min-php-version-will-be-81" >}}).
+
+**Symfony 5.4**
+
+Yes! Symfony has been upgraded from version 4.4 to [version 5.4](https://symfony.com/releases/5.4). It's one of the [long-term supported versions](https://symfony.com/doc/current/contributing/community/releases.html#maintenance). As with every new version of Symfony, there are some [new features](https://symfony.com/blog/category/living-on-the-edge/6.0-5.4?page=1) available for developers, as well as breaking changes. Right now, we [list all the notable changes in PrestaShop 9](https://github.com/PrestaShop/docs/issues/1658), so that we can later add them into the developer documentation.
+
+If you are interesting in how the upgrade process looked like, you can see [this pull request](https://github.com/PrestaShop/PrestaShop/pull/31216).
+
+**New default theme (?)**
+
+A few months ago [we informed about Hummingbird]({{< relref "/news/posts/2022/new-theme-announce" >}}). Since then, a lot has changed in the theme. Currently, work on the theme continues [based on the list of issues](https://github.com/orgs/PrestaShop/projects/11) that were split into those that are a must-have for the first test version, nice-to-haves, and so on. We will need your help testing and making the default theme for PrestaShop as best as possible. If you are a developer and want to discuss with us the future of front-end in PrestaShop, make sure to visit us on the [project's Slack](https://www.prestashop-project.org/slack/), `#hummingbird-theme` channel. It hasn't been decided yet whether `Hummingbird` will replace Classic or will be an alternative theme. We'll keep you updated.
```

Comment:
```suggestion
A few months ago [we informed you on a Build blog post about Hummingbird]({{< relref "/news/posts/2022/new-theme-announce" >}}). Since then, a lot has changed on the theme. Currently, work on the theme continues [based on the list of issues](https://github.com/orgs/PrestaShop/projects/11) that were split into categories: must-have for the first test version, nice-to-have, and so on. We will need your help testing and making the default theme for PrestaShop as best as possible. If you are a developer and want to discuss with us the future of front-end in PrestaShop, make sure to visit us on the [project's Slack](https://www.prestashop-project.org/slack/), `#hummingbird-theme` channel. It hasn't been decided yet whether `Hummingbird` will replace Classic or will be an alternative theme. We'll keep you updated.
```